chapters/3-package-information.md

```diff
@@ -523,38 +673,45 @@ Example:
 
 Example:
 
-    PackageChecksum: SHA1: 85ed0817af83a24ad8da68c2b5094de69833983c
+```text
+PackageChecksum: SHA1: 85ed0817af83a24ad8da68c2b5094de69833983c
+```
+```text
```

Comment:
Does MarkDown need a line break between these two sections? I wasn't sure but I see there's an additional empty line for the other examples (e.g. between SHA256 and MD5 below).